programs/loft.sc

```diff
@@ -0,0 +1,356 @@
+__config() -> {
+    'stay_loaded'->true,
+    'commands'->{
+        'wand'->['wand'],
+        'paste <Block> <replace>'->['paste'],
+        'paste <Block>'->['paste',null],
+        'paste'->['paste','diamond_block',null],
+        'undo'->['undo']
+    },
+    'arguments'->{
+        'Block'->{'type'->'term', 'suggest'->[
+			'white_concrete','yellow_concrete','orange_concrete','purple_concrete',
+			'lime_concrete','red_concrete','green_concrete','magenta_concrete',
+			'brown_concrete','gray_concrete','light_gray_concrete','blue_concrete',
+			'light_blue_concrete','pink_concrete','cyan_concrete','black_concrete'
+			]
+		},
+        'replace'->{'type'->'term','suggest'->['air','water','lava']}
```

Comment:
Well, as suggested on discord, move these two over to an actual block type, so that the minecraft command parser can handle stuff like `dirp` suggest autocompletions for blocks that are not concrete.

Reply:
Ok, I'll try to moove the pull request in the right section.
About the surface selection, you have to create 2 or more splines to have a proper surface. As said in the script, left click starts a new spline and right click expands the last spline created. Then after you have selected at least 2 splines, the paste command should be successfull. The script basically just do a loft using those splines created by the player and that generates a curve surface. If there is only one spline the paste will fail. It will olso fail if you create more splines but each of them is composed by only one point. There is a way to paste a single spline and that is to select 2 splines using the same control points. So the first spline is exactly equal to the second one.
Hope this explenation will help